site/includes/brand.hbs

```diff
@@ -1,6 +1,7 @@
 {{#unless provisional}}
 	<div class="brand col-xs-5 col-md-4" property="publisher" typeof="GovernmentOrganization">
-		<a href="https://www.canada.ca/{{language}}.html" property="url"><img src="{{assets}}/../{{site.theme}}/assets/sig-blk-{{language}}.svg" alt="" property="logo" /><span class="wb-inv" property="name"> {{{i18n "tmpl-gc-sig"}}} / <span lang="{{#is page.language "en"}}fr">{{{i18n "tmpl-gc-sig" language="fr"}}}{{else}}en">{{{i18n "tmpl-gc-sig" language="en"}}}{{/is}}</span></span></a>
+		<a href="https://www.canada.ca/{{language}}.html" property="url"><img src="{{assets}}/../{{site.theme}}/assets/sig-blk-{{language}}.svg" alt="{{{i18n "tmpl-gc-sig"}}}" property="logo" /><span class="wb-inv"> / <span lang="{{#is page.language "en"}}fr">{{{i18n "tmpl-gc-sig" language="fr"}}}{{else}}en">{{{i18n "tmpl-gc-sig" language="en"}}}{{/is}}</span></span></a>
```

Comment:
Is the "\" need to be excluded from the language change? can we simply have 1 span instead of 2?
Like:
```
<span class="wb-inv" lang="{{#is page.language "en"}}fr"> / {{{i18n "tmpl-gc-sig" language="fr"}}}{{else}}en">{{{i18n "tmpl-gc-sig" language="en"}}}{{/is}}</span></a>
```
The slash is only use as a language change text marker.

What is your thought?

Reply:
#noslah

Reply:
@bsouster what do you think if we remove the "/" character in the screen reader only text and keep the language change? like
```
 property="logo" /><span class="wb-inv" lang="{{#is page.language "en"}}fr"> {{{i18n "tmpl-gc-sig" language="fr"}}}{{else}}en">{{{i18n "tmpl-gc-sig" language="en"}}}{{/is}}</span></a>
```

Reply:
@bsouster Any thoughts on this?